data_hub_api/docmaps/sql/docmaps_index.sql

```diff
@@ -202,7 +191,53 @@ t_result AS (
   WHERE
     Version.Overall_Stage = 'Full Submission'
     AND Version.Position_In_Overall_Stage = 1
-    AND COALESCE(preprint_doi_and_url.preprint_doi, biorxiv_medrxiv_response.doi) IS NOT NULL
+),
+
+t_result_with_preprint_dois AS (
+  SELECT 
+    * EXCEPT(ejp_normalized_title)
+  FROM t_initial_result
+  WHERE t_initial_result.preprint_doi IS NOT NULL
+),
+
+t_result_for_partially_match_title AS (
+  SELECT
+    t_initial_result.* EXCEPT(ejp_normalized_title, preprint_doi, preprint_doi_source),
+    biorxiv_medrxiv_response.doi AS preprint_doi,
+    'biorxiv_medrxiv_title_match_partial' AS preprint_doi_source,
+
+  FROM t_biorxiv_medrxiv_response_by_normalized_title AS biorxiv_medrxiv_response
+  INNER JOIN t_initial_result
+    ON biorxiv_medrxiv_response.normalized_title LIKE CONCAT('%', t_initial_result.ejp_normalized_title, '%')
```

Comment:
Is this just matching `13-12-2022-RA-RP-eLife-85562` or are there other manuscripts that are matched by this?

What is the expected behaviour if this was matching more than one manuscript?

We have seen that doing the partial matching higher up caused it to take too long. This is now filtering by `-RP-` and currently many of them are not matched (for an unknown reason). As we are just getting started with reviewed preprints, could this soon run into a similar problem?

Reply:
Just for reference, the other match with the title would match with the first (based on `imported_timestamp`) bioRxiv paper that has the same title.
(Because `t_biorxiv_medrxiv_response_by_normalized_title` is partitioned by the normalized title, ordered by `imported_timestamp` ascending)